using_images/other_images/jenkins.adoc

```diff
@@ -52,18 +52,104 @@ You can find https://github.com/openshift/origin/tree/master/examples/image-stre
+* hudson.model.Item.Build
+* hudson.model.Item.Configure
+* hudson.model.Item.Create
+* hudson.model.Item.Delete
+* hudson.model.Item.Read
+* hudson.model.Item.Workspace
+* hudson.model.Run.Delete
+* hudson.model.Run.Update
+* hudson.model.View.Configure
+* hudson.model.View.Create
+* hudson.model.View.Delete
+* hudson.scm.SCM.Tag
+
+Users with the `edit` or `view``*Roles*` for the {product-title} project have progressively reduced permissions within Jenkins.
+
+For the `view` `*Role*`:
+
+* hudson.model.Hudson.Read
+* hudson.model.Item.Read
+
+For the `edit` `*Role*`, in addition to the permissions available to `*view*`:
+
+* hudson.model.Item.Build
+* hudson.model.Item.Configure
+* hudson.model.Item.Create
+* hudson.model.Item.Delete
+* hudson.model.Item.Workspace
+* hudson.scm.SCM.Tag
+
+Users authenticated against {product-title} OAuth will be added to the Jenkins authorization matrix upon their first successful login.
+
+Permissions for users in Jenkins can be changed in {product-title} after those users are initially established in Jenkins.  The OpenShift Login plugin polls the {product-title} API server for permissions and will update the permissions stored in
+Jenkins for each Jenkins user with the permissions retrieved from {product-title}.  If the Jenkins UI is used to update permissions for a Jenkins user, the permission changes will be overwritten the next time the plugin polls {product-title}.
+
+You can control how often the polling occurs with the `OPENSHIFT_PERMISSIONS_POLL_INTERVAL` environment variable.  The default polling interval when no environment variable is set is 5 minutes.
+
+The following command creates a new Jenkins xref:../../architecture/core_concepts/pods_and_services.adoc#pods[pod] using {product-title} OAuth authentication:
+
+----
+$ oc new-app -e \
+    OPENSHIFT_ENABLE_OAUTH=true \
+    openshift/jenkins-1-centos7
+----
+
+==== Jenkins standard authentication
+
+Jenkins authentication will be used by default with the {product-title} Jenkins images if the image is run outside of {product-title}.
+
 The first time you start Jenkins, the configuration is created along with the administrator user and password.
-The default login is `*admin/password*`.
+The default user login is `*admin/password*`.
 The default password can be configured by setting the `*JENKINS_PASSWORD*` environment variable.
 
-The following command creates a new Jenkins xref:../../architecture/core_concepts/pods_and_services.adoc#pods[pod] with Jenkins running in a container:
+The following command creates a new Jenkins xref:../../architecture/core_concepts/pods_and_services.adoc#pods[pod] using standard Jenkins authentication:
 
 ----
 $ oc new-app -e \
     JENKINS_PASSWORD=<password> \
+    OPENSHIFT_ENABLE_OAUTH=false \
     openshift/jenkins-1-centos7
 ----
 
+If you choose to run with standard Jenkins authentication while the {product-title} Jenkins image runs inside of {product-title}, and you instantiate Jenkins from the template
```

Comment:
OK pushed update that has command invocations around instantiating templates in the oauth section, and and the oc new-app against the image for the standard auth.  Added clarifications the JENKINS_PASSWORD is only applicable to standard jenkins auth.